server/helpers/constants.js

```diff
@@ -120,3 +120,9 @@ export const ACTIVE_PORTFOLIO_OFFER = [
 ];
 
 export const REFERRAL_PERCENTAGE = 1.5;
+
+export const BADGE_ACCESS_LEVEL = {
+  ALL: 0,
```

Comment:
Since we can't have a role of `-1`, I think it would be a better alternative to avoid using the same role value as an admin.
```suggestion
  ALL: -1,
```